client/packages/system/src/Item/Components/ColdStorageTypeInput/ColdStorageTypeInput.tsx

```diff
@@ -28,18 +30,18 @@ export const ColdStorageTypeInput = ({
   return (
     <Autocomplete
       disabled={disabled}
-      clearable={false}
       value={buffer && { ...buffer, label: buffer.name }}
       loading={isLoading}
       onChange={(_, name) => {
         setBuffer(name);
-        name && onChange(name);
+        onChange(name);
       }}
       options={data?.coldStorageTypes.nodes ?? []}
       getOptionLabel={option => option.name}
       width={`${width}px`}
       popperMinWidth={width}
       isOptionEqualToValue={(option, value) => option?.id === value?.id}
+      inputProps={{ label }}
```

Comment:
Will probably remove this when we update the modal UI as I outlined in PR description